commerce-payment-method-money-order/src/main/resources/META-INF/resources/configuration.jsp

```diff
@@ -0,0 +1,63 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+MoneyOrderGroupServiceConfiguration moneyOrderGroupServiceConfiguration = (MoneyOrderGroupServiceConfiguration)request.getAttribute(MoneyOrderGroupServiceConfiguration.class.getName());
+
+String messageXml = null;
+
+LocalizedValuesMap messageLocalizedValuesMap = moneyOrderGroupServiceConfiguration.message();
+
+if (messageLocalizedValuesMap != null) {
+	messageXml = LocalizationUtil.getXml(messageLocalizedValuesMap, "message");
+}
+%>
+
+<portlet:actionURL name="editMoneyOrderCommercePaymentMethodConfiguration" var="editCommercePaymentMethodActionURL" />
+
+<aui:form action="<%= editCommercePaymentMethodActionURL %>" cssClass="container-fluid-1280" method="post" name="fm" onSubmit='<%= "event.preventDefault(); " + renderResponse.getNamespace() + "saveCommercePaymentMethod();" %>'>
```

Comment:
I stick to how other forms in the Commerce have been done 😄

Reply:
Ha. Alright. I'll change it here since it will act exactly the same (only faster) to remove the javascript. (We should probably look elsewhere in our code sometime to make sure we only use javascript to submit when necessary.)